modules/claims_api/app/services/claims_api/power_of_attorney_request_service/terminate_existing_requests.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'bgs_service/veteran_representative_service'
+require 'bgs_service/manage_representative_service'
+
+module ClaimsApi
+  module PowerOfAttorneyRequestService
+    class TerminateExistingRequests
+      DEFAULT_FIRST_NAME = 'vets-api'
```

Comment:
We don't really have a "user" taking an action in the way SEP does when it updates statuses based on representative actions. There is test data using "VDC User" as a name, so using "vets-api" to keep moving.

Reply:
I think we can loop back on adding a "Veteran User" if we find the need. for now, to keep us moving I'm fine with continuing this flow until decided.